### ISSUE TITLE
`koji-osbuild` is missing from the developer guides

### ISSUE THREAD

**Issue by supakeen** [osbuild/osbuild.github.io#80]
The projects overview in the developer guides *should* include [koji-osbuild](https://github.com/osbuild/koji-osbuild).

**Comment by ochosi** [comment]
True. Hadn't pulled that in because we're mostly handling it with the Koji maintainers directly. I don't expect a wider audience to take an interest in this - would you?
(Also want to avoid to clutter the development section too much, which is why more projects from GitHub aren't there at this point..)
